lib/Test/MockFile.pm

```diff
@@ -708,6 +698,10 @@ sub _abs_path_to_file {
     my ($path) = shift;
 
     defined $path or return;
+
+    # cleanup multiple slashes
+    $path =~ s{[/]+}{/}xmsg;
```

Comment:
micro optimization: maybe `//+` ? to avoid replacing all single `/`

Reply:
Done.